src/KeyVault/KeyVault/Commands/Certificate/ImportAzureKeyVaultCertificate.cs

```diff
@@ -101,20 +116,65 @@ public class ImportAzureKeyVaultCertificate : KeyVaultCmdletBase
         /// Password
         /// </summary>
         [Parameter(Mandatory = false,
-                   ParameterSetName = ImportCertificateFromFileParameterSet,
+                   ParameterSetName = ImportCertificateFromFileWithPolicyParameterSet,
+                   HelpMessage = "Specifies the password for the certificate and private key file to import.")]
+        [Parameter(Mandatory = false,
+                    ParameterSetName = ImportWithPrivateKeyFromStringWithPolicyParameterSet,
+                    HelpMessage = "Specifies the password for the certificate and private key base64 encoded string to import.")]
+        [Parameter(Mandatory = false,
+                    ParameterSetName = ImportWithPrivateKeyFromCollectionWithPolicyParameterSet,
+                    HelpMessage = "Specifies the password for the certificate collection and private key to import.")]
+        [Parameter(Mandatory = false,
+                   ParameterSetName = ImportCertificateFromFileWithPolicyFileParameterSet,
                    HelpMessage = "Specifies the password for the certificate and private key file to import.")]
         [Parameter(Mandatory = false,
-                    ParameterSetName = ImportWithPrivateKeyFromStringParameterSet,
+                    ParameterSetName = ImportWithPrivateKeyFromStringWithPolicyFileParameterSet,
                     HelpMessage = "Specifies the password for the certificate and private key base64 encoded string to import.")]
+        [Parameter(Mandatory = false,
+                    ParameterSetName = ImportWithPrivateKeyFromCollectionWithPolicyFileParameterSet,
+                    HelpMessage = "Specifies the password for the certificate collection and private key to import.")]
         public SecureString Password { get; set; }
 
+        /// <summary>
+        /// File Path
+        /// </summary>
+        [Parameter(Mandatory = false,
+                   ParameterSetName = ImportCertificateFromFileWithPolicyFileParameterSet,
+                   HelpMessage = "Specifies the path to the file that contains the certificate policy to import to key vault.")]
```

Comment:
```suggestion
                   HelpMessage = "A file path to specify management policy for the certificate that contains JSON encoded policy definition. ")]
```
Please refer to https://github.com/Azure/azure-rest-api-specs/blob/e716082ac474f182e2220e4f38f1d6191e7636cf/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.4/certificates.json#L1855